x/market/handler/endblock.go

```diff
@@ -55,6 +56,49 @@ func transferFundsForActiveLeases(ctx sdk.Context, keepers Keepers) error {
 	return nil
 }
 
+func pickBidWinner(bids []types.Bid) (winner types.Bid) {
+	// open bids; match by lowest price; sort bids by price
+	sort.Slice(bids, func(i, j int) bool {
+		// AFAICT: the BidID DSeq is pulled from the original OrderID.
+		// So it can't be used to determine who bid first.
```

Comment:
yeah, all of these bids should be for the same order (same order id -> same group id -> same deployment id).

Reply:
update comments